README.md

```diff
@@ -1,36 +1,42 @@
 # [Squoosh]!
 
-[Squoosh] is an image compression web app that allows you to dive into the advanced options provided
-by various image compressors.
+[Squoosh] is an image compression web app that provides lossless image quality with a significant reduction to file size.
 
 # API & CLI
 
-Squoosh now has [an API](https://github.com/GoogleChromeLabs/squoosh/tree/dev/libsquoosh) and [a CLI](https://github.com/GoogleChromeLabs/squoosh/tree/dev/cli) that allows you to compress many images at once.
+Squoosh has [an API](https://github.com/GoogleChromeLabs/squoosh/tree/dev/libsquoosh) and [a CLI](https://github.com/GoogleChromeLabs/squoosh/tree/dev/cli) to compress many images at once.
 
 # Privacy
 
-Google Analytics is used to record the following:
+Squoosh does not send your image to a server. All image compression processes locally.
 
-- [Basic visit data](https://support.google.com/analytics/answer/6004245?ref_topic=2919631).
-- Before and after image size once an image is downloaded. These values are rounded to the nearest
-  kilobyte.
-- If install is available, when Squoosh is installed, and what method was used to install Squoosh.
+However, Squoosh utilizes Google Analytics to collect the following:
 
-Image compression is handled locally; no additional data is sent to the server.
+- [Basic visitor data](https://support.google.com/analytics/answer/6004245?ref_topic=2919631).
+- The before and after image size value.
+- If Squoosh PWA, the type of Squoosh installation.
+- If Squoosh PWA, the installation time and date.
 
-# Building locally
+# Developing
 
-Clone the repo, and:
+To develop for Squoosh:
 
-```sh
-npm install
-npm run build
-```
+1. Clone the repository
+1. To install node packages, run:
+   ```sh
+   npm install
+   ```
+1. Then build the app by running:
+   ```sh
+   npm run build
+   ```
+1. After building, start the development server by running:
+   ```sh
+   npm run dev
+   ```
```

Comment:
```suggestion
1. Clone the repository
2. To install node packages, run:
   ```sh
   npm install
   ```
3. Then build the app by running:
   ```sh
   npm run build
   ```
4. After building, start the development server by running:
   ```sh
   npm run dev
   ```
```

Reply:
Hi @nucliweb, 

Please see https://github.com/mikedelgaudio/squoosh/tree/update-readme I believe markdown supports an automatic scalable number increase by using `1.` Also https://www.markdownguide.org/basic-syntax/#ordered-lists

<img width="903" alt="Screen Shot 2021-07-12 at 2 36 23 PM" src="https://user-images.githubusercontent.com/43451174/125338956-d1019d00-e31e-11eb-9aa8-8d6444f68939.png">

Reply:
Ok, I think that is more semantic code with incremental numbers 😉
But it's fine